azure-pipelines-release.yml

```diff
@@ -7,18 +7,15 @@ pool:
   demands:
     - ImageOverride -equals MMS2022TLS
 
+variables:
+  project: 'src/dirs.proj'
+
 steps:
 - checkout: self
   submodules: true
 
 - task: UseDotNet@2
-  displayName: 'Use the .NET Core 2.1 SDK (required for build signing)'
```

Comment:
Was the ESPR step improved to not require .NET Core 2.1 anymore?

Reply:
Yup! The `@2` version now uses 3.1, 5, or 6.